src/Nri/Ui/Checkbox/V5.elm

```diff
@@ -443,8 +458,15 @@ checkboxChecked =
         |> Nri.Ui.Svg.V1.fromHtml
 
 
-checkboxCheckedPartially : Svg
-checkboxCheckedPartially =
+checkboxCheckedPartially : String -> Svg
+checkboxCheckedPartially idSuffix =
+    let
+        filterId =
+            "filter-2" ++ idSuffix
```

Comment:
🎨 I guess we could `String.Extra.dasherize` this to make it safer in case the id contains spaces? I don't think that's likely to happen though so totally optional.